docs/tests/__snapshots__/props-table.test.ts.snap

```diff
@@ -11552,55 +11552,6 @@ exports[`Props Table 1`] = `
                         \\"description\\": \\"Changes the underlying rendered HTML element\\",
                         \\"category\\": \\"BaseComponentProps\\",
                         \\"isOptional\\": true
-                    },
-                    \\"BaseTextFieldProps\\": {
-                        \\"name\\": \\"BaseTextFieldProps\\",
-                        \\"type\\": \\"extends TextFieldOptions, BaseInputProps\\",
-                        \\"description\\": \\"\\",
-                        \\"category\\": \\"TextField\\",
-                        \\"isOptional\\": true
-                    },
-                    \\"resize\\": {
```

Comment:
This prop and a few others are from `TextArea`. It should not be included in `TextField` anymore.